templates/bootstrap_rpm/rhsm-katello-reconfigure.erb

```diff
@@ -1,10 +1,10 @@
 KATELLO_SERVER=<%= @rhsm_hostname %>
 KATELLO_SERVER_CA_CERT=<%= @server_ca_name %>.pem
 KATELLO_DEFAULT_CA_CERT=<%= @default_ca_name %>.pem
-KATELLO_CERT_DIR=<%= @rhsm_ca_dir %>
 PORT=<%= @rhsm_port %>
 
-PREFIX=<%= @rhsm_prefix %>
+CERT_DIR='/etc/rhsm/ca'
+PREFIX=/rhsm
```

Comment:
I actually would like to keep this a variable for now. The `/rhsm` will be used in the Apache config as well and when it's a variable, you can see it's related. Even though it's unlikely to change. This is because Puppet doesn't really have constants, but constants(/variables) are a really good way of indicating related systems.

Reply:
The /rhsm is not used in the Apache config today:

```
# grep -r rhsm /etc/httpd/
```

As the reverse proxy proxies all traffic back to Foreman and Katello defines a `/rhsm` API. So there is nothing to relate it back to it from what I can tell. This just "is".